### PR TITLE
Remove 3.x changelog

### PR DESCRIPTION
**Pull request by koppor**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -271,717 +271,15 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
...
- - Fixed bug when having added and then removed a personal journal list, an exception is always shown on startup
- - Fixed a bug in the IEEEXploreFetcher
- - Fixed [bug 1282](https://sourceforge.net/p/jabref/bugs/1282/) related to backslashes duplication.
- - Fixed [bug 1285](https://sourceforge.net/p/jabref/bugs/1285/): Editing position is not lost on saving
- - Fixed [bug 1297](https://sourceforge.net/p/jabref/bugs/1297/): No console message on closing
- - Fixed [#194](https://github.com/JabRef/jabref/issues/194): JabRef starts again on Win XP and Win Vista
- - Fixed: Tooltips are now shown for the #-field when the bibtex entry is incomplete.
- - Fixed [#173](https://github.com/JabRef/jabref/issues/173): Personal journal abbreviation list is not loaded twice
- - Bugfix: Preview of external journal abbreviation list now displays the correct list
- - Fixed [#223](https://github.com/JabRef/jabref/issues/223): Window is displayed in visible area even when having multiple screens
```
Comment (stefan-kolb) [inline CHANGELOG.md]: We need such a text for 3.x too

Comment (koppor) [reply CHANGELOG.md]: Which text? 😇 

The 3.0 changelog points back to the 2.11 one. But not forward to 4.0 though...

**Comment by tobiasdiez** [comment]
Why don't we just keep the changelog of the 3.* versions in the same file? I always liked that one could easily search in the changelog.

**Comment by lenhard** [comment]
From the devcall: we decided to merge this and remove the old changelog entries.

**Pull request merged**